k2/python/k2/fsa_algo.py

```diff
@@ -26,7 +25,9 @@ def linear_fsa(symbols: Union[List[int], List[List[int]]]) -> Fsa:
       A vector of FSAs if the input is a list of list of integers.
     '''
     ragged_arc = _k2.linear_fsa(symbols)
-    return Fsa.from_ragged_arc(ragged_arc)
+    fsa = Fsa.from_ragged_arc(ragged_arc)
+    fsa.arc_labels_sorted = True
```

Comment:
My instinct here is to just compute and cache the properties and not treat `arc_labels_sorted` specially.
Although we can of course provide a convenient interface to test individual properties.
It may be sufficient to always deal with the scalar properties for now; we can add the vector properties as and when we need them.

Reply:
An attribute `properties` is added. Users can pre-compute the properties of an FSA and cache it before calling
any fsa algorithms.